docker/images/outrigger/README.md

```diff
@@ -0,0 +1,53 @@
+# Outrigger Docker Image
+
+# Building
+
+To build simply run:
+
+```
+docker build -t outrigger .
+``` 
+
+in the directory this README is currently in.
+
+# Running
+
+To run the outrigger server simply run:
+
+```
+docker run -d -p 8080:8080 -v `pwd`/juttles:/opt/juttles --name outrigger outrigger
+```
+
+The above exposes the 2 ports that are necessary to talk between your local
+browser and the outriggerd server and also exposes the juttles directory 
+found here for demo purposes so you can then use the following link to run
+the "Hello World" juttle program:
+
+Now hit the following URL in your browser:
+
+    http://localhost:8080/run?path=/opt/juttles/hello_world.juttle
```

Comment:
I'll add the Mac vs Linux different in here (obviously didn't think about how on mac/windows this all runs in a VM).